Lib/os.py

```diff
@@ -55,6 +55,7 @@ def _get_exports_list(module):
     from posix import *
     try:
         from posix import _exit
+        from posix import _get_cpu_count_config
```

Comment:
This means it is only possible to override cpu_count on posix?

Reply:
Reasonable!